docs/installing-solidity.rst

```diff
@@ -210,12 +210,30 @@ You will need to install the following dependencies for Windows builds of Solidi
 +------------------------------+-------------------------------------------------------+
 | `CMake`_                     | Cross-platform build file generator.                  |
 +------------------------------+-------------------------------------------------------+
-| `Visual Studio 2015`_        | C++ compiler and dev environment.                     |
+| `Visual Studio 2017 Build Tools`_ | C++ compiler                                     |
 +------------------------------+-------------------------------------------------------+
+| `Visual Studio 2017`_  (Optional) | C++ compiler and dev environment.                |
++------------------------------+-------------------------------------------------------+
+
+Visual Studio 2017 Build Tools provides all required compiler and libraries.
+If you prefer your own IDE, Visual Studio 2017 Build Tools is a perfect choice.
```

Comment:
I'm not sure if we are at liberty to say that it is a "perfect choice".

Could you please tone down this and the next paragraph so that we just suggest this as an IDE, but do not praise it? :)